home/templates/home/tags/footer.html

```diff
@@ -1,27 +1,11 @@
 {% load static wagtailcore_tags wagtailimages_tags %}
 <footer class="footer">
     <nav>
-        {% for block in footer.navigation %}
         <div class="links">
```

Comment:
We should probably move this template one level up since this is not going to be a template tag anymore

Reply:
as we don't pass data to it from context but fetch data before rendering and include it as a self-contained tag so we don't need to move it.